cms/models.py

```diff
@@ -115,7 +182,7 @@ class Meta:
         FieldPanel("what_you_learn"),
         FieldPanel("feature_image"),
     ]
-    parent_page_types = ["HomePage"]
+
```

Comment:
I don't see the need for this change. We can keep `ProductPage`/`CoursePage` as a child page of `HomePage` until we implement the course index page properly. If there's no immediate need to change that hierarchy, we don't need this change as part of this PR

Reply:
@gavin if we move the product page to child of just  homepage then while creating Homepage in CMS you cannot linked products then so good to keep on page level right now.